--- navit-0.1.0/navit/maps/Makefile.am	2009-01-05 16:03:54.000000000 -0800
+++ navit-0.1.0/navit/maps/Makefile.am.new	2009-01-05 16:04:09.000000000 -0800
@@ -6,11 +6,6 @@
 
 samplemap: $(SAMPLE_MAP).bin
 
-$(SAMPLE_MAP).osm.bz2:
-	echo "Downloading osm sample map"
-	wget -O $(SAMPLE_MAP).osm.bz2.tmp http://www.navit-project.org/maps/$(SAMPLE_MAP).osm.bz2
-	mv $(SAMPLE_MAP).osm.bz2.tmp $(SAMPLE_MAP).osm.bz2
-
 $(SAMPLE_MAP).bin: $(SAMPLE_MAP).osm.bz2 $(top_builddir)/navit/osm2navit
 	echo "Converting osm sample map"
 	bzcat $(SAMPLE_MAP).osm.bz2 | $(top_builddir)/navit/osm2navit --attr-debug-level=5 $(SAMPLE_MAP).bin.tmp
