--- navit-0.1.0/configure.in	2008-11-26 13:42:14.000000000 -0800
+++ navit-0.1.0/configure.in.new	2009-01-04 22:25:41.000000000 -0800
@@ -596,9 +596,9 @@
   if test "$XGETTEXT" != "" -a "$HAVEGETTEXT" != ""; then
      PO=""
      if test "$LINGUAS" = ""; then
-           ling=` (cd $srcdir/po; /bin/ls *.po.in) `
+           ling=` (cd $srcdir/po; /bin/ls *.po) `
   	   for l in $ling; do
-	        lcode=`basename $l .po.in`
+	        lcode=`basename $l .po`
 	        LINGUAS="$LINGUAS$lcode "
            done
     fi
--- navit-0.1.0/po/Makefile.am	2008-11-26 13:42:14.000000000 -0800
+++ navit-0.1.0/po/Makefile.am.new	2009-01-04 22:31:08.000000000 -0800
@@ -28,7 +28,7 @@
 all: @MOFILES@
 
 @POFILES@: navit.pot
-	sed 's/^"Project-Id-Version:.*/"Project-Id-Version: $(PACKAGE_STRING)\\n"/' < $(top_srcdir)/po/$@.in | $(MSGMERGE) --no-fuzzy-matching --verbose -o $(top_builddir)/po/$@ - navit.pot
+#	sed 's/^"Project-Id-Version:.*/"Project-Id-Version: $(PACKAGE_STRING)\\n"/' < $(top_srcdir)/po/$@.in | $(MSGMERGE) --no-fuzzy-matching --verbose -o $(top_builddir)/po/$@ - navit.pot
 
 navit.pot: $(POTFILES)
 	 $(XGETTEXT) --default-domain=navit \
